inc/cachify.class.php

```diff
@@ -1605,8 +1605,8 @@ private static function _minify_select()
 	{
 		return array(
 			self::MINIFY_DISABLED  => esc_html__('No minify', 'cachify'),
-			self::MINIFY_HTML_ONLY => 'HTML',
-			self::MINIFY_HTML_JS   => 'HTML + Inline JavaScript'
+			self::MINIFY_HTML_ONLY => esc_html__('HTML', 'cachify'),
```

Comment:
Please, I will buy a blank for `esc_html__( 'HTML', 'cachify' ),`, before and after brackets.